plugin/plugin.go

```diff
@@ -184,10 +194,14 @@ type Signature struct {
 // CriticalAttributes contains all Notary V2 defined critical
 // attributes and their values in the signature envelope
 type CriticalAttributes struct {
-	ContentType        string                 `json:"contentType"`
-	SigningScheme      string                 `json:"signingScheme"`
-	Expiry             *time.Time             `json:"expiry,omitempty"`
-	ExtendedAttributes map[string]interface{} `json:"extendedAttributes,omitempty"`
+	ContentType                  string                 `json:"contentType"`
+	SigningScheme                string                 `json:"signingScheme"`
+	Expiry                       *time.Time             `json:"expiry,omitempty"`
+	SigningTime                  *time.Time             `json:"signingTime,omitempty"`
```

Comment:
SigningTIme is not a critical attribute as per signature spec, we can remove this. I added it mistakenly in the plugin spec, which is not corrected - https://github.com/notaryproject/notaryproject/pull/184/files